data.sql

```diff
@@ -16,4 +16,50 @@ VALUES
   (8, 'Angemon', '2005-06-12', 1, true, -45),
   (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
   (10, 'Blossom', '1998-10-13', 3, true, 17),
-  (11, 'Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+  (11, 'Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners(full_name, age)
+VALUES
+('Sam Smith', 34 ),
+('Jennifer Orwell', 19 ),
+('Bob ', 45 ),
+('Melody Pond', 14 ),
+('Melody Pond', 14 ),
+('Jodie Whittaker', 38 );
+
+INSERT INTO species(name)
+VALUES
+('Pokemon'),
+('Digimon');
+
+
+UPDATE animals AS a
+SET species_id = s.id
+FROM species AS s
+WHERE a.name LIKE '%mon%' AND s.name = 'Digimon';
```

Comment:
- [ ] Nice writing the SQL code :clap: However we want to set all animals that it's names **end** with `mon`. But your code will select all animals with `mon` in the middle of their name. Please take the example of this SQL code to understand how this would be done. 
```sql
 UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```